template/CONTRIBUTING.md.jinja

```diff
@@ -15,7 +15,7 @@ You can contribute in many ways, for example:
 
 ### Report Bugs
 
-Report bugs at {{ dev_platform_url }}/{{ username }}/{{ project_slug }}/issues.
+Report bugs at {{ dev_platform_url }}{{ username }}/{{ project_slug }}/issues.
```

Comment:
By removing the manual `/` the url renders correctly. I am actually not sure where the slash comes from in this template as i don't see it defined in the copier.yml template for dev_platform_url!